beamer/tests/agent/multiple_processors/test_slave.py

```diff
@@ -0,0 +1,32 @@
+import json
+import os
+import sys
+import time
+import ape
+
+from beamer.tests.util import alloc_accounts, make_request
+
+
+def test_sending_request_slave(contracts, token, finality_period):
+    requester, target = alloc_accounts(2)
+    chain_id = ape.chain.chain_id
+    request_count = int(os.environ.get("REQUEST_COUNT", 0))
```

Comment:
This environment variable must be present, right?
In that case, I would not call `get` -- let it crash the test.